Extension/src/LanguageServer/client.ts

```diff
@@ -1017,9 +1018,12 @@ class DefaultClient implements Client {
             this.model.isTagParsing.Value = true;
             testHook.updateStatus(Status.TagParsingBegun);
         } else if (message.endsWith("Updating IntelliSense...")) {
+            timeStamp = Date.now();
             this.model.isUpdatingIntelliSense.Value = true;
             testHook.updateStatus(Status.IntelliSenseCompiling);
         } else if (message.endsWith("IntelliSense Ready")) {
+            let duration: number = Date.now() - timeStamp;
+            console.log("Flame time (sec):", duration / 1000);
```

Comment:
This should only be logged when the loggingLevel is "Debug".
Consider also rewording this to "Refresh IntelliSense time".

Reply:
I think it should be "Update IntelliSense time" to match the hover message, right?

Reply:
Done